test/production/app-dir/metadata-img-too-large/opengraph-image/index.test.ts

```diff
@@ -11,11 +11,13 @@ describe('app-dir - metadata-img-too-large opengraph-image', () => {
 
   it('should throw when opengraph-image file size exceeds 8MB', async () => {
     await next.patchFile('app/opengraph-image.png', pngFile as any)
-
     await next.build()
-    const { cliOutput } = next
-    expect(cliOutput).toMatch(
-      /Error: File size for Open Graph image ".*\/app\/opengraph-image\.png" exceeds 8MB/
-    )
+
+    const regex = isTurbopack
+      ? // in Turbopack, the path is simplified as [project]/...
```

Comment:
shouldn't this regex (that was there originally) match both?
`/Error: File size for Open Graph image ".*\/app\/opengraph-image\.png" exceeds 8MB/`

Reply:
Right, just improved it that we confirm it outputs `[project]/` for turbopack.

Reply:
not sure if that's necessary but i don't mind either way really